index.html

```diff
@@ -54,12 +54,25 @@
   <section id='abstract' data-include="abstract/common-abstract.md" data-include-format='markdown'></section>
   <section id='sotd' data-include="abstract/common-sotd.md" data-include-format='markdown'></section>
 
-  <section id='conformance'></section>  
+  <section id='conformance'></section>
 
   <section data-include="abstract/baggage-overview.md" data-include-format='markdown'></section>
+  <section data-include="abstract/privacy.md" data-include-format='markdown'></section>
 
   <section data-include="baggage/HTTP_HEADER_FORMAT.md" data-include-format='markdown'></section>
 
 </body>
 
+  <section class="informative">
+    <h1>Glossary</h1>
+    <dl>
+    <dt><dfn data-lt="opaque">Opaque value</dfn></dt>
+    <dd>
+      An opaque value refers to a value that can only be understood
+      or processed in any way by the distributed trace participant that generated
+      this value. Any other participant must treat it as a blob of bytes.
```

Comment:
> participant that generated this value

this is incorrect, or subject to how "participant" is defined (the usual meaning won't work here). While the value is opaque, it is meant to transmit information to _another participant_.

Reply:
fixed